iped-utils/src/main/java/iped/utils/fsw/FileAttributes.java

```diff
@@ -0,0 +1,69 @@
+package iped.utils.fsw;
+
+import java.nio.file.Files;
+import java.nio.file.NotDirectoryException;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.nio.file.attribute.FileTime;
+
+public class FileAttributes implements BasicFileAttributes {
```

Comment:
@patrickdalla seems many methods of this class are not forwarding the calls to the wrapped Path, shouldn't it forward all of them?